app/assets/stylesheets/components/_error-summary.scss

```diff
@@ -0,0 +1,72 @@
+@import "helpers/contents-list-helper";
+
+.app-c-error-summary {
+  color: $app-text-colour;
+  padding: $app-spacing-scale-3;
+
+  border: $app-border-width-mobile solid $app-error-colour;
+
+  @include media(tablet) {
+
+    padding: $app-spacing-scale-4;
+
+    border: $app-border-width-tablet solid $app-error-colour;
+  }
+}
+
+.app-c-error-summary:focus {
+  outline: $app-focus-width solid $app-focus-colour;
+}
+
+.app-c-error-summary__title {
+  margin-top: 0;
+  margin-bottom: $app-spacing-scale-3;
+
+  @include media(tablet) {
+    margin-bottom: $app-spacing-scale-4;
+  }
+
+  @include bold-24;
+}
+
+.app-c-error-summary__body {
+  @include core-19;
+}
+
+.app-c-error-summary__text {
+  margin-top: 0;
+  margin-bottom: $app-spacing-scale-3;
+
+  @include media(tablet) {
+    margin-bottom: $app-spacing-scale-4;
+  }
+}
+
+.app-c-error-summary__list {
+  margin-top: 0;
+  margin-bottom: 0;
+  list-style: none;
+}
+
+.app-c-error-summary__link {
+  display: inline-block;
```

Comment:
I think originally I put extra padding on these links, then decided against it since the focus states looked very odd, I'll see if I can rearrange this.